app/controllers/kudos_controller.rb

```diff
@@ -3,6 +3,7 @@
 class KudosController < ApplicationController
   before_action :set_kudo, only: %i[show edit update destroy]
   before_action :authenticate_employee!
+  before_action :correct_employee, only: %i[edit update destroy]
```

Comment:
[FYI] - actually no-one is using custom callbacks nowadays - they are making the controller code much harder to read, but let's leave them for now - you will refactor it eventually anyways ;)

Reply:
so i should code this checks into methods?